admin/local/docker/compose/docker-compose.yml

```diff
@@ -348,11 +354,13 @@ services:
           entrypoint --log-level DEBUG replication-controller
           --db-uri mysql://qsreplica@repl-mgr-db:3306/qservReplica
           --db-admin-uri mysql://root:CHANGEME@repl-mgr-db:3306/qservReplica
-          --qserv-czar-db mysql://root:CHANGEME@czar-db:3306/qservMeta
-          --instance-id qserv_proj
           --worker qserv_worker_db=mysql://qsmaster@worker-db-0:3306/qservw_worker,host=worker-repl-0
           --worker qserv_worker_db=mysql://qsmaster@worker-db-1:3306/qservw_worker,host=worker-repl-1
           --xrootd-manager manager-xrootd
+          --
+          --instance-id=qserv_proj
+          --qserv-czar-db=mysql://root:CHANGEME@czar-db:3306/qservMeta
```

Comment:
This option (`--qserv-czar-db=`) is de-facto mandatory in all Qserv deployments because it's used by the Controller to update `czar`'s database during catalog ingests. I would recommend keeping it in the list of the entry point's options (before`--`).

Reply:
`entrypoint` does not need it, and we'd have to reproduce the help and so forth. Isn't this inconsistent with what you've said about `entrypoint` not getting in-between the CLI and the options used by the repl binary? We should discuss this during cowork?

Reply:
The idea that I had in mind here was to build (or amend) that URI by the entry point based on the security tokens (password, etc.) injected into the containers via Kubernetes config-maps, security, or environment variables, or some other mechanism. The same applies to all database connectors.
Let's discuss this topic at the group meeting!